docs/source/api.rst

```diff
@@ -38,6 +38,9 @@ Dataset Generation (Dask-based Multi-GPU)
   .. automodule:: cuml.dask.datasets.blobs
      :members:
 
+  .. automodule:: cuml.dask.datasets.regression
+     :members:
+
```

Comment:
Wondering if we want to include our new dask mixins w/ the BaseEstimator somewhere in the docs for users to get familiar w/ the class/mixin hierarchies

Reply:
Also, I've started a directory `python/cuml/prims` with some initial primitives that seem to be getting a lot of use lately. Do we also want to include these in the docs?

Reply:
I'm adding the mixins but note that their methods are all private so they won't show up.